src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
+import java.util.Random;
 
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+       int index = new Random().nextInt(Colors.values().length);
+       String res = String.valueOf(Colors.values()[index]);
+       return res;
```

Comment:
Don’t create temporary variables if you can directly return value from the method
```suggestion
       return String.valueOf(Colors.values()[index]);
```